app/routes/resources+/user-event.cjs

```diff
@@ -0,0 +1,2 @@
+// userEvent types are missing when import as default to ESM module
+export { default as userEvent } from '@testing-library/user-event'
```

Comment:
This will create a route at `/resources/user-event` which we definitely don't want. Can we find another way to do this? Maybe open an issue on user event?

Reply:
how about we use `// @ts-expect-error` for now

I can try to create a .d.ts file for '@testing-library/user-event'

Reply:
I've moved it out of resources